tools/RAiDER/cli/raider.py

```diff
@@ -245,12 +246,27 @@ def calcDelays(iargs=None):
         # Grab the closest two times unless the user specifies 'nearest'
         # If the model time_delta is not specified then use 6
         # The two datetimes will be combined to a single file and processed
-        times = get_nearest_wmtimes(t, [model.dtime() if \
-                    model.dtime() is not None else 6][0]) if params['interpolate_time'] else [t]
+        interp_method = params['interpolate_time']
```

Comment:
@bbuzz31 - when you read a yaml file with a given field and it is not specified - then he returns None so the default behavior is not changed. I will change this to `get` for further clarity.